src/mnemonic/mnemonic.js

```diff
@@ -30,7 +28,8 @@ var unorm = require('unorm');
  * @returns {Mnemonic} A new instance of Mnemonic
  * @constructor
  */
-var Mnemonic = function(data, wordlist) {
+// eslint-disable-next-line consistent-return
+const Mnemonic = function (data, wordlist) {
   if (!(this instanceof Mnemonic)) {
```

Comment:
This is a construct that makes it possible to create a new object without calling new. I think this is an anti-pattern and we should remove this construct from every class (in a separate pull request)

Reply:
Follow-up #59